spec/views/spotlight/shared/_configuration_sidebar.html.erb

```diff
@@ -0,0 +1,18 @@
+<h4 class="nav-heading"><%=t(:'spotlight.configuration.sidebar.header') %></h4>
```

Comment:
To add the `Services` link below.  FWIW, this is all being merged into a feature branch, so we have time to change direction on this.  This is one thing I was thinking we could do something interesting in Spotlight to accommodate, but our timeframe to complete this work has been severely limited.